./run_api_tests.sh: line 93: python: command not found
